TezosKit/Conseil/Transaction.swift

```diff
@@ -0,0 +1,39 @@
+// Copyright Keefer Taylor, 2019
+
+import Foundation
+
+/// A transaction retrieved from Conseil.
+public struct Transaction {
```

Comment:
This covers transactions reasonably well; you may want to add `Invocation` as a type as well and expose `parameters` there.